run_tjpcov.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python
+
+from tjpcov.main import CovarianceCalculator
+
+
+if __name__ == "__main__":
```

Comment:
Do you think, instead of having this separated file, we could write this execution script in the main.py?

Reply:
Sure. I find it a bit more confusing but nothing critical. The real reason to put it outside is that i didn't want to modify things were already there to avoid breaking the tests.